Team6220_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team6220_CENTERSTAGE/ColorDetectionPipeline.java

```diff
@@ -0,0 +1,88 @@
+package org.firstinspires.ftc.team6220_CENTERSTAGE;
+
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfPoint;
+import org.opencv.core.Point;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.core.Size;
+import org.opencv.imgproc.Imgproc;
+import org.openftc.easyopencv.OpenCvPipeline;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ColorDetectionPipeline extends OpenCvPipeline
+{
+    boolean viewportPaused = false;
+    Mat input = new Mat();
+    Mat threshold = new Mat();
+    Mat contourMask = new Mat();
+    Mat output = new Mat();
+    Scalar minRange = new Scalar(0, 0, 0);
```

Comment:
Can this be in the Constants class?

Reply:
They can't because they are object-specific. I did remove two unnecessary ones, however